Marlin/Conditionals_LCD.h

```diff
@@ -101,7 +110,8 @@
   #if ENABLED(ULTIMAKERCONTROLLER)              \
    || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) \
    || ENABLED(G3D_PANEL)                        \
-   || ENABLED(RIGIDBOT_PANEL)
+   || ENABLED(RIGIDBOT_PANEL)                   \
+   || ENABLED(REPRAPWORLD_KEYPAD)
```

Comment:
I'm in favor of NOT changing how the default REPRAPWORLD_KEYPAD is implemented.  I'm thinking that @benlye's approach is the best way.

Should NEWPANEL also be included?  That's the other item that the REPRAPWORLD_KEYPAD change picked up.
```
  #if ENABLED(ANET_KEYPAD_LCD)
     #define ULTIPANEL
    #define NEWPANEL
    ...
  #endif
```